.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -185,7 +185,6 @@ def __call__(self, parser, namespace, values, option_string=None):
     'vcs_url': '',
     'authentication_method': 0,
     'institution_name' : '',
-    'username_change_text' : 'Submitty welcomes individuals of all ages, backgrounds, citizenships, disabilities, sex, education, ethnicities, family statuses, genders, gender identities, geographical locations, languages, military experience, political views, races, religions, sexual orientations, socioeconomic statuses, and work experiences. In an effort to create an inclusive environment, you may specify a preferred name to be used instead of what was provided on the registration roster.',
```

Comment:
I may not be up to date with the conversion, but was there a reason this is removed as a configurable parameter for the hardcoded one?

Reply:
Barb asked me to